pkgs/applications/networking/mailreaders/caeml/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "caeml";
+  version = "unstable-2023-05-24";
+
+  src = fetchFromGitHub {
+    owner = "ferdinandyb";
+    repo = pname;
```

Comment:
nitpick: 
```suggestion
    repo = finalAttrs.pname;
```
and remove rec

Reply:
looks like `buildGoModule` doesn't support the `finalAttrs` pattern yet :/

Reply:
Yeah sorry, didn't know that :see_no_evil: